cmd/kube-apiserver/app/preflight/checks_test.go

```diff
@@ -31,7 +31,7 @@ func TestParseServerURIGood(t *testing.T) {
 
 	reference := "127.0.0.1:2379"
 	if host != reference {
-		t.Fatal("server uri was not parsed correctly")
+		t.Fatal("server uri was not parsed correctly, host %s was invalid", host)
```

Comment:
Actually, this doesn't pass lint.
You need to change to "Fatalf" now (instead of "Fatal").